band_together/entities/player/player.gd

```diff
@@ -60,6 +60,12 @@ func _physics_process(delta):
 	gravity(delta)  # Slow fall, wall slide, jump buffer, coyote time is also in here!
 	jump(delta)  # All types of jumps (wall jump, double jump, etc!)
 	move_and_animate()  # Sets velocity based on input, changes sprites
+	
+	if is_on_floor():
+		GameManager.save_ground_position(global_position)  # Store last ground position
+	
+	if global_position.y > 1000:  # TODO: Refractor to calculate WorldBoundary. If the player falls out of the world boundary, respawn
```

Comment:
Temporary, but hopefully we can either be confident about the world boundary limits, or try to calculate it.